logger.py

```diff
@@ -115,7 +119,6 @@ def restore_logger_object(self, log):
         self.rc_event_schedule_initial = log["rc_event_schedule_initial"]
         self.rc_event_damage_initial = log["rc_event_damage_initial"]
         self.number_riskmodels = log["number_riskmodels"]
-        del log["rc_event_schedule_initial"], log["rc_event_damage_initial"], log["number_riskmodels"]
```

Comment:
For the ensemble run, these must be either removed or `rc_event_damage_initial` must be transformed from `np.ndarray` to list type. Otherwise the log will include code (`'{..., rc_event_damage_initial': [[], [array([0.26523393])], [array([0.26076461]), array([0.33464024])], []], ...}`) that cannot be interpreted by `eval()` in `visualisation.py`.

Reply:
The schedule method of Setup file has been altered so that it doesn't write array() to the file, just the number, as I initially had this exact issue.